fsw/src/crypto_props.c

```diff
@@ -0,0 +1,351 @@
+/*
+ This implementation file implements functions from crypto_props.h
+ * which are required for key,value property management via linked list
+ * data structure.  
+ */
+/* 
+ * File:   crypto_props.c
+ * Author: Ary Naim (aryan.e.naim@jpl.nasa.gov)
+ *
+ * Created on November 4, 2021, 9:37 AM
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include "stdbool.h"
```

Comment:
Why is "stdbool.h" included instead of <stdbool.h>?

Reply:
updated. These new changes will in the new pull request.